superset-frontend/src/utils/hostNamesConfig.js

```diff
@@ -16,15 +16,24 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import {
+  initFeatureFlags,
+  isFeatureEnabled,
+  FeatureFlag,
+} from 'src/featureFlags';
+
 function getDomainsConfig() {
   const appContainer = document.getElementById('app');
   if (!appContainer) {
     return [];
   }
 
   const bootstrapData = JSON.parse(appContainer.getAttribute('data-bootstrap'));
+  // eslint-disable-next-line camelcase
+  initFeatureFlags(bootstrapData?.common?.feature_flags);
```

Comment:
this module (sometimes?) is loaded before App.jsx, it cause feature_flags is empty.

Reply:
yikes... when does that happen?

Reply:
i am not sure...most of places, when we isFeatureEnabled, is after initFeatureFlags (which was called in index.jsx). Maybe this module is a little special, currently it is loaded before index.jsx. I found this issue only when i am working on this PR.
But it's webpack decide which module loaded first.

Reply:
fair enough. maybe add a comment explaining this as well just in case